meinberlin/templates/header.html

```diff
@@ -3,36 +3,51 @@
 <header class="main-header">
     <div class="l-wrapper">
         <nav class="main-nav">
-            <img class="beberlin-logo" src="{% static 'images/beberlin.svg' %}" height="22" width="90" alt="be Berlin" />
-            <a href="/" class="main-nav__logo" rel="home">
-                <img src="{% static 'images/LOGO.png' %}" alt="mein.berlin"/>
-            </a>
-            <button class="btn btn--light main-nav__toggle" data-toggle="collapse" data-target="#menu-collapse" aria-expanded="false" aria-controls="menu-collapse">
-                <i class="fa fa-bars" aria-hidden="true"></i>
-                {% trans 'Menu' %}
-            </button>
-            <div class="main-nav__menu-wrapper collapse" id="menu-collapse">
-                <ul class="main-nav__list">
-                    <li class="main-nav__item">
-                        <a class="main-nav__link" href="{% url 'project-list' %}">{% trans 'Participation projects' %}</a>
-                    </li>
-                    {% get_menu "topnav" as topnav %}
-                    {% for item in topnav %}
+            <div class="main-nav__nav-item">
+                <button class="btn btn--light main-nav__toggle" data-toggle="collapse" data-target="#menu-collapse" aria-expanded="false" aria-controls="menu-collapse" aria-label="{% trans 'Menu' %}">
+                    <i class="fas fa-bars" aria-hidden="true"></i>
+                </button>
+                <div class="main-nav__menu-wrapper collapse" id="menu-collapse">
+                    <ul class="main-nav__list">
                         <li class="main-nav__item">
-                            <a class="main-nav__link" href="{{ item.url }}">{{ item.title }}</a>
+                            <a class="main-nav__link" href="/"><i class="fas fa-home"></i></a>
                         </li>
-                    {% endfor %}
-                </ul>
-                <div class="main-header__user">
-                    {% userindicator %}
+                        <li class="main-nav__item">
+                            <a class="main-nav__link" href="{% url 'project-list' %}">{% trans 'Where can I take part?' %}</a>
+                        </li>
+                        {% get_menu "topnav" as topnav %}
+                        {% for item in topnav %}
+                            <li class="main-nav__item">
+                                <a class="main-nav__link" href="{{ item.url }}">{{ item.title }}</a>
+                            </li>
+                        {% endfor %}
+                    </ul>
+                </div>
+            </div>
+            <div class="main-nav__logo-item">
+                <a href="/" class="main-nav__logo" rel="home">
+                    <img src="{% static 'images/LOGO.png' %}" alt="mein.berlin"/>
+                    <div class="main-nav__headline">{% trans 'Participation platform of Berlin!' %}</div>
```

Comment:
As the image next to it has to be centered, this has (?) to be positioned absolute, but I just can't get it vertically centered.